internal/policy/policy.go

```diff
@@ -366,6 +367,63 @@ func (s *State) FindPublicKeysForPath(ctx context.Context, path string) ([]*tuf.
 	}
 }
 
+func (s *State) FindVerifiersForPath(ctx context.Context, path string) ([]*VerificationContext, error) {
```

Comment:
This will replace `FindPublicKeysForPath` once we have the verification context behavior nailed down for all scenarios. Right now, for example, we need to see how it affects `verifyTagEntry` where all keys from a policy are used if no specific trusted keys are found.